### DISCUSSION TITLE
To edit (Add or Delete) reciters and translations.

### DISCUSSION THREAD

**Discussion by Koushik-9119** [AlfaazPlus/QuranApp#375] [General]
assalamu alaikum wa rahmatullahi wa barakatuh sir 
when i modify the **reciters, translations and Tafsir** **available_recitation_translations_info.json** & **available_recitations_info.json** in inventory/recitations/
while application is running on android, it is showing like this.

![Screenshot_20230813-185204_QuranApp Debug](https://github.com/AlfaazPlus/QuranApp/assets/126067987/b772482f-6d78-478d-a9c3-1c4594c16838)
sir how to edit **(Add or Delete) ** **reciters, translations and Tafsir** these information in this application.

![Screenshot_20230813-183251_QuranApp Debug](https://github.com/AlfaazPlus/QuranApp/assets/126067987/76f69295-85c2-4029-8bd8-5cf6121d68b0)
![Screenshot_20230801-234426_QuranApp](https://github.com/AlfaazPlus/QuranApp/assets/126067987/53bf4554-87d9-4129-aaa6-316534faaa67)

**Comment by faisalcodes** [comment]
Walaikum assalam
If you want to contribute a new reciter, please post the information here. If you want to modify it for personal use only, then you'll yourself have to figure out how to do it.
